project/src/js/ui/friendsPopup.js

```diff
@@ -37,28 +37,48 @@ friendsPopup.view = function() {
 };
 
 function renderFriends() {
+
   return friendsApi.list().length ?
     (
-    <ul>
-      {friendsApi.list().map(renderFriend)}
-    </ul>
+        <ul>
+          {friendsApi.list().map(renderFriend)}
+        </ul>
     ) : (
       <div className="native_scroller nofriend">{i18n('noFriendsOnline')}</div>
     );
 }
 
-function renderFriend(name) {
-  let userId = utils.userFullNameToId(name);
+function renderFriend(user) {
+
+  // hack: Because we have an 'oncreate' within an 'oncreate' two events
```

Comment:
You should stop the event propagation instead:

```js
function onTvTap(e: event) {
   e.stopPropagation();
   // do stuff
}
```

or... you can attach a single event handler that would handle both TV and User taps (using `event.target` as discriminant`)